tutorial/practicals/LFRic/single_node/code/scaled_matrix_vector_kernel_mod.F90

```diff
@@ -45,8 +45,7 @@ module scaled_matrix_vector_kernel_mod
 
 use argument_mod,            only : arg_type,                 &
                                     GH_FIELD, GH_OPERATOR,    &
-                                    GH_READ, GH_INC,          &
-                                    ANY_SPACE_1, ANY_SPACE_2, &
+                                    GH_REAL, GH_READ, GH_INC, &
```

Comment:
Aren't ANY_SPACE_1/2 still needed?

Reply:
Oh yes, I must have accidentally deleted this line. I combed through other incorrect instances of this kernel and there was one in `examples/lfric/eg11` so I corrected it here.